docs/meetings/hpc/2022-12-07.md

```diff
@@ -0,0 +1,82 @@
+---
+tags: meeting, notes
+---
+
+# JupyterHub HPC Meeting - December 2022
+
+- **Date:** 2022-12-07
+- **Time:** 8:30 AM PST
+  - **Your timezone:** https://arewemeetingyet.com/Los%20Angeles/2022-12-07/08:30/JupyterHub-HPC
+- **Where you can find these and past meeting notes:** https://jupyterhub-team-compass.readthedocs.io/en/latest/meetings/hpc/ 
+- **Calendar for future meetings:** https://jupyterhub-team-compass.readthedocs.io/en/latest/meetings/
+
+## Welcome to the Meeting
+
+Hello! If you are joining the team video meeting, sign in below so we know who was here. Roll call:
+
+- **name** / **institution** / **GitHub handle**
+- Michael Milligan / MSI @ UMN / @mbmilligan
+- Jens Henrik Goebbert / JSC @ FZJ / @jhgoebbert
+- Rollin Thomas / NERSC / @rcthomas
+- 
+
+## Quick updates
+
+60 second updates on things you have been up to, questions you have, or developments you think people should know about. This is also a chance to suggest a future presentation if you've got work currently in progress you might want to share. Please add yourself, and if you do not have an update to share, you can pass.
+
+- **Name:** Your report or celebration
+    - Jens Henrik Göbbert
+        - SLURM Wrapped Kernels (by Tim Kreuzer)
+            - https://github.com/FZJ-JSC/jupyter-slurm-provisioner
```

Comment:
https://github.com/FZJ-JSC/jupyter-slurm-provisioner goes to a 404 page, is it private?

Reply:
I think this is a typo or a duplication, because the line afterward is the right one I believe.  I'll fix.

Reply:
Thanks @aagrosch!   Fixed and clarified.